com.palantir.typescript/src/com/palantir/typescript/IPreferenceConstants.java

```diff
@@ -79,4 +80,7 @@
     String SYNTAX_COLORING_PUNCTUATION_COLOR = "syntaxColoring.punctuation.color";
     String SYNTAX_COLORING_REG_EXP_LITERAL_COLOR = "syntaxColoring.regExpLiteral.color";
     String SYNTAX_COLORING_STRING_LITERAL_COLOR = "syntaxColoring.stringLiteral.color";
+
+    String PREFERENCE_STORE_TS_CONFIG_LAST_MODIFICATION_TIME = "preferenceStore.tsConfigLastModTime";
```

Comment:
These constants look like they are alpha-ed. Both within blocks and across blocks.